gr-utils/python/modtool/core/add.py

```diff
@@ -127,10 +129,10 @@ def _write_tpl(self, tpl, path, fname):
 
     def run(self):
         """ Go, go, go. """
-        # This portion will be covered by the CLI
-        if not self.cli:
```

Comment:
In my opinion, we should not validate the fields that are already verified through the CLI. Rather, we should raise similar exception in cli version of `add.py` after taking the input for language.

Reply:
Don't want to brush you off @swap-nil7, but we need to move forward on this PR and its harmless enough.